src/components/Home.vue

```diff
@@ -57,32 +60,53 @@ export default {
   data() {
     return {
       selectedUser: '',
-      showModalWallet: false
+      moneyTransfer: '',
+      showWalletModal: false,
+      showSendingMoneyModal: false,
     };
   },
   computed: {
-    userName() {
-      return this.$store.getters.updateUserName;
+    loginUser() {
+      return this.$store.getters.getLoginUser;
     },
     users() {
-      return this.$store.getters.getUser;
-    }
+      return this.$store.getters.getUsers;
+    },
   },
   // リロードしてもユーザー名が消えないようにする処理
   created() {
-    this.$store.dispatch('updateUserName');
+    this.$store.dispatch('getLoginUser');
+    this.$store.dispatch('getUsers');
   },
   methods: {
-    async logOut() {
-      await this.$store.dispatch('logOut');
-      this.$router.push('/');
+    logOut() {
+      this.$store.dispatch('logOut');
+        this.$router.push('/');
     },
     selectUser(user) {
       this.selectedUser = user;
-      this.showModalWallet = true;
+      this.showWalletModal = true;
+    },
+    //送るボタンをクリックされたユーザー情報をselectedUserに格納する為の関数
+    usersSendMoney(user) {
+      this.selectedUser = user;
+      this.showSendingMoneyModal = true;
     },
     closeModal() {
-      this.showModalWallet = false;
+      this.showWalletModal = false;
+      this.showSendingMoneyModal = false;
+    },
+    //入力された金額分だけ投げ銭する関数
+    async sendMoney() {
+      if(this.moneyTransfer === '') {
+        return
+      } else {
+        await this.$store.dispatch('updateBalance', {user: this.selectedUser, money: this.moneyTransfer});
```

Comment:
elseを削除しても同じ挙動になる場合、elseは削除して良いです。
そして空文字判定なら
`if(!this.moneyTransfer)`
とでき、さらにreturnするコードが一行で書けるなら{}も省略できます。
まとめると以下の感じです。
```suggestion
      if(!this.moneyTransfer) return
      await this.$store.dispatch('updateBalance', {user: this.selectedUser, money: this.moneyTransfer});
```